CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -33,7 +33,7 @@ local RobloxGui = CoreGui:FindFirstChild("RobloxGui")
 local ContextActionService = game:GetService("ContextActionService")
 
 -- Enable the old Utility.lua if the EnablePortraitMode flag is off
-local enablePortraitModeSuccess, enablePortraitModeValue = pcall(function() return settings():GetFFlag("EnablePortraitMode") end)
+local enablePortraitModeSuccess, enablePortraitModeValue = pcall(function() return UserSettings():IsUserFeatureEnabled("EnablePortraitMode") end)
```

Comment:
I am not 100% positive changing this to IsUserFeatureEnabled works, lets keep these as they are